scripts/db-tunnel.sh

```
@@ -0,0 +1,171 @@
+echo ""
+echo "Option 3: Manual SSM port forwarding (if you have an EC2 instance)"
+echo "  aws ssm start-session --target <instance-id> \\"
+echo "    --document-name AWS-StartPortForwardingSessionToRemoteHost \\"
+echo "    --parameters '{\"host\":[\"$DB_HOST\"],\"portNumber\":[\"5432\"],\"localPortNumber\":[\"5432\"]}'"
+echo ""
+echo "Once connected, source credentials with:"
+echo "  source $TEMP_FILE"
+echo ""
+
+# Keep temp file available for manual use
+read -p "Press Enter to exit (temp credentials file will be deleted)..."
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)